script.service.singealarmclock/resources/lib/tests/test_alarmclock.py

```diff
@@ -0,0 +1,141 @@
+# pylint: disable=missing-docstring,invalid-name,import-error
```

Comment:
please remove `/tests` from the submission. They are useful for development but not for distribution.